boards/acd52832/doc.txt

```diff
@@ -0,0 +1 @@
+ */
```

Comment:
Looks like a bug: your script is creating a doc.txt when the board is not documented in the code base and not present in the wiki. I saw a few others like this.

Reply:
Yes, it's a bug. I will fix it when I'm back

Reply:
fixed